learners/src/components/display.js

```diff
@@ -0,0 +1,8 @@
+class Display extends React.Component {
```

Comment:
Who created this file and any modifications you did please mention who did and date of modification